examples/taint-checking/taint_checking.description

```diff
@@ -1,8 +1,12 @@
-// Ballerina is designed to ensure that programs written in Ballerina are inherently secure. Ballerina programs
-// are resilient to major security vulnerabilities including SQL injection, path manipulation, file manipulation,
-// unauthorized file access, and unvalidated redirect (open redirect).<br/><br/>
+// Ballerina is designed to ensure that programs written in Ballerina are inherently secure.
+// With taint flow analysis (a.k.a taint checking) Ballerina compiler identifies data flows which may lead to
```

Comment:
```suggestion
// With taint flow analysis the Ballerina compiler identifies data flows which may lead to
```

Reply:
Do we need to specifically mention "taint checking"?

Reply:
Although the process is taint flow analysis, the flag and the toml property to enable it is `taint check`, thought it would be helpful to have some reference to taint checking in the description.